src/main/java/appeng/api/stacks/AEKey.java

```diff
@@ -209,6 +209,8 @@ public final boolean matches(@Nullable GenericStack stack) {
 
     public abstract String getModId();
 
+    public abstract String getId();
```

Comment:
This needs to be documented. What kind of ID is this? Does it include the prefix, or doesn't it? Is it the translation key?